src/app/contexts/ToggleContext/index.test.jsx

```diff
@@ -1,214 +1,48 @@
 import React, { useContext } from 'react';
 import { render, act } from '@testing-library/react';
-import fetchMock from 'fetch-mock';
 import '@testing-library/jest-dom/extend-expect';
 import { queryByText } from '@testing-library/dom';
-import togglesConfig from '#lib/config/toggles';
 import { ToggleContext, ToggleContextProvider } from '.';
 
 // eslint-disable-next-line react/prop-types
 const TestComponent = ({ toggle, children }) => {
-  const toggleIsEnabled = useContext(ToggleContext).toggleState[toggle].enabled;
-
-  return toggleIsEnabled && <div>{children}</div>;
+  const { toggleState } = useContext(ToggleContext);
+  const toggleIsEnabled = toggleState[toggle] && toggleState[toggle].enabled;
+  if (!toggleIsEnabled) {
+    return null;
+  }
+  return <div>{children}</div>;
 };
 
-const shouldCallTogglesEndpoint = togglesUrl =>
-  expect(fetchMock.calls(togglesUrl).length).toBeTruthy();
-const shouldNotCallTogglesEndpoint = togglesUrl =>
-  expect(fetchMock.calls(togglesUrl).length).toBeFalsy();
-const shouldRenderAd = container =>
-  expect(queryByText(container, 'Dummy Ad Component')).toBeInTheDocument();
-const shouldNotRenderAd = container =>
-  expect(queryByText(container, 'Dummy Ad Component')).not.toBeInTheDocument();
+const mockRemoteToggles = {
+  testToggle: {
+    enabled: true,
```

Comment:
Maybe we should add a test case where the `testToggle` is `enabled: false`?

Reply:
This exists below, can't recall if we added that in after the comment :)